docs/config/api.rst

```diff
@@ -136,9 +136,12 @@ Memspace
 
 TODO: Add general information about memspaces.
 
+.. note::
+   The memspace, memtarget and mempolicy APIs are experimental and may change in future releases.
```

Comment:
this should be repeated in every section - one might scroll down to memtarget and not see this message at all

Reply:
Done